### PR TITLE
[wip] DDNS tutorial edits

### PR DESCRIPTION
**Pull request by matthewrevell**
This is an edited version of the dynamic DNS tutorial submitted by jodumont.

At this stage I'm looking for confirmation that we're happy with what the tutorial recommends.

I've created a new fork in order to preserve jodumont's commit history and make my changes without first merging jodumount's branch into master.

### CODE REVIEW

File: community_tutorials/2016-07-11-ddns-api-recipe.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+Now we need to do some work on the machine that want to reference with our sub-domain.
+
+### Find the record id
+
+Each of your domain's records has a unique ID. We'll need that ID to set the sub-domain's IP address using the Exoscale API.
+
+Here's a simple script that pulls down the domain's record information:
+
+	#!/bin/bash
+	#The best I could it's this ugly script
```
Comment (marcaurele) [inline community_tutorials/2016-07-11-ddns-api-recipe.md]: Some left over? ;-)

File: community_tutorials/2016-07-11-ddns-api-recipe.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+	Secret="___ YOUR SECRET KEY ___"
+	DOMAIN_NAME="___ YOUR DOMAIN NAME ___"
+
+	curl -H "X-DNS-Token: $API:$Secret" -H "Accept: application/json" "https://api.exoscale.ch/dns/v1/domains/$DOMAIN_NAME/records"
+
+Find your API Key and Secret [in your Exoscale console](https://portal.exoscale.ch/account/profile/api).
+
+Make it executable:
+
+	$ chmod 700 /root/bin/getid_exodns.sh
```
Comment (marcaurele) [inline community_tutorials/2016-07-11-ddns-api-recipe.md]: I wouldn't put the `/root/bin` prefix. Best not to run _internet_ scripts as root, and the user can save this script where he wants. I would put `$ chmod a+x getid_exodns.sh`

File: community_tutorials/2016-07-11-ddns-api-recipe.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+Then run it:
+
+	$ /root/bin/getid_exodns.sh
```
Comment (marcaurele) [inline community_tutorials/2016-07-11-ddns-api-recipe.md]: Same here: ~~`/root/bin/`~~ but `$ ./getid_exodns.sh`

**Comment by marcaurele** [comment]
No command requires the script to be run as root, so I would leave this choice to the reader.

**Comment by ste-m** [comment]
Hi,
@ste-m from Exoscale here.
I'm sorry to communicate that we decided to close the contribution project.
We thank you for the time you've spent on this PR but we will now close it.

**Pull request closed without merging**